packages/core/src/components/Alert/Alert.example.jsx

```diff
@@ -3,19 +3,16 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 
 ReactDOM.render(
-  <div>
+  <div className="example--wrapper">
```

Comment:
Is `example--wrapper` to be used on all examples or only specific ones?

Reply:
It only works for certain examples like Badge. This is because components like `TextField` are composed of 2 adjacent elements, and `example--wrapper` will put the margin on both elements unless we wrap the elements with a div, and I didnt want to clutter up our code examples.

Theres other cases too where theres already margin from a label or some other child element, so it doesnt make sense to use `example--wrapper`.

This definetely isnt perfect... I wish the margins on our components were consistently on the outer wrapper so that these examples could be spaced perfectly